01-getting-started/installation/platforms/install-on-gcp.md

```diff
@@ -0,0 +1,195 @@
+
+2. Register by clicking **Sign up with Google** (2) and sign-in using your existing Google Cloud credentials. This is the fastest way to register.
+
+![Create MongoDB Step 2](/images/started/gcp/2022-10-20_07-37.png)
+
+3. On successful registration and login you will be redirected to cloud.mongodb.com portal _Projects_ page.
+4. Start by creating a project. Click **New Project** (3).
+
+![Create MongoDB Step 3](/images/started/gcp/2022-10-20_07-46.png)
+
+5. Provide a project name such as _Squidex_ (4) and click **Next** (5) to continue.
+
+![Create MongoDB Step 4](/images/started/gcp/2022-10-20_07-51.png)
+
+6. Finish project creation by clicking **Create Project** (6).
+
+![Create MongoDB Step 5](/images/started/gcp/2022-10-20_07-53.png)
+
+7. Next step is to create the MongoDB database. Click **Build a Database** (7) to start.
+
+![Create MongoDB Step 6](/images/started/gcp/2022-10-20_07-56.png)
+
+8. Select **Create** under _Shared_ which provides 1 free cluster.
+
+![Create MongoDB Step 7](/images/started/gcp/2022-10-20_07-58.png)
+
+9. Ensure again that **Free Shared** (9) option is selected, click **Google Cloud** (10), select a region of choice (11) and click **Create Cluster** (12).
+
+![Create MongoDB Step 8](/images/started/gcp/2022-10-20_08-31.png)
+
+10. In the next step, set Security details. Enter a Username (13) and Password (14) of choice (or click Autogenerate Secure Password to get one generated). Click **Create User** (15).
+
+![Create MongoDB Step 10](/images/started/gcp/2022-10-20_08-12.png)
+
+11. Finally, add network access permissions so Squidex can talk to it. Ensure **My Local Environment** (16) is selected, enter 0.0.0.0/0 (17) for IP Address and any Description (18). Click **Add Entry** (19).
+
+![Create MongoDB Step 11](/images/started/gcp/2022-10-20_08-15.png)
+
+12. Click **Finish and Close**. This will redirect you to the Database Deployments page similar to the screenshot below.
+
+![Create MongoDB Step 12](/images/started/gcp/2022-10-20_12-45.png)
+
+13. You will need the connection information for this MongoDB for use in the app configuration later. To copy the connection string click **Connect** (20) and then select **Connect your application** (21).
+
+![Create MongoDB Step 13](/images/started/gcp/2022-10-20_12-41.png)
+
+14. Select C#/.NET (22) & 2.13 or later (23), copy the connection string (24) and click **Close** (24)
+
+![Create MongoDB Step 14](/images/started/gcp/2022-10-20_12-44.png)
+
+This completes creating the free MongoDB Atlas cluster.
+
+## 2. Upload Squidex image to Artifact Registry
+
+The following steps require ![gcloud CLI](https://cloud.google.com/sdk/docs/install) which can be installed on your development machine or you can use the Cloud Shell. The later is preferred for convenience. To launch Cloud Shell click ![Cloud Shell icon](/images/started/gcp/2022-10-20_18-24.png) on the google cloud web console next to the search bar.
```

Comment:
I think this is not true, I upload docker tags from CI without the CLI. You just need a keyfile.

Reply:
What does CI mean (I know only of continuous integration)? Google Artifact Registry replaces google cloud registry and does not have an upload option from UI.
We can mention this is one of the ways and state something like "for this article we will use gcloud CLI".

Reply:
Yes, continuous integration, basically "docker push".

Reply:
Yes we are using docker push command, but we also need to run some gcloud commands to enable apis, create resources etc. So running all of them from Cloud Shell is an easier approach.